convenience-stores-jp/src/main/java/com/adyen/checkout/conveniencestoresjp/ConvenienceStoresJPConfiguration.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ozgur on 20/1/2023.
+ */
+
+package com.adyen.checkout.conveniencestoresjp
+
+import android.content.Context
+import com.adyen.checkout.action.GenericActionConfiguration
+import com.adyen.checkout.components.model.payments.Amount
+import com.adyen.checkout.core.api.Environment
+import com.adyen.checkout.econtext.EContextConfiguration
+import kotlinx.parcelize.Parcelize
+import java.util.Locale
+
+@Suppress("LongParameterList")
+@Parcelize
+class ConvenienceStoresJPConfiguration private constructor(
```

Comment:
Don't forget to add methods for these configurations in the `DropInConfiguration.Builder`